.gitignore

```diff
@@ -2,7 +2,7 @@
 *~
 *.diff
 .*.swp
-minmea
-tests
-example
+minmea/tests
+minmea/example
 *.exe
+build
```

Comment:
This is not necessary:
- it may be named anything and a popular convention is `builddir/`, or `builddir-release/` and `builddir-debug/`, so this gitignore entry likely won't take effect
- Meson knows at the time of creating `builddir/` that it is a build directory, and that no files in there should be added to git, so it creates `builddir/.gitignore` which ignores `*`, so that you don't have to. This just magically works.

  Admittedly, this was implemented in December 2020 via https://github.com/mesonbuild/meson/commit/7437d9862a16bab3fd0c025769d5928f2ed1548e and released in Meson 0.57.0, so back when you originally submitted this PR that was not the case, but still... it is long past December 2020, so you can remove this now. :)